src/views/main-view.js

```diff
@@ -170,7 +170,8 @@ MainView.prototype.setPrimaryView = function (id, secondaryViewId) {
 
   this.model.setPaymentMethodRequestable({
     isRequestable: Boolean(paymentMethod),
-    type: paymentMethod && paymentMethod.type
+    type: paymentMethod && paymentMethod.type,
+    selectedPaymentMethod: paymentMethod
```

Comment:
I elected to send along the `selectedPaymentMethod` here, rather than doing the `isPaymentMethodSelected` logic here. Is that 👍  with everyone?